src/android/src/com/chariotsolutions/nfc/plugin/NfcPlugin.java

```diff
@@ -622,10 +650,14 @@ public void run() {
                     for (String tagTech : tag.getTechList()) {
                         Log.d(TAG, tagTech);
                         if (tagTech.equals(NdefFormatable.class.getName())) {
-                            fireNdefFormatableEvent(tag);
+                            //fireNdefFormatableEvent(tag);
```

Comment:
what's the reason for this being commented out? either keep, remove or add a comment on why it is commented out